genesyscloud/telephony_providers_edges_extension_pool/resource_genesyscloud_telephony_providers_edges_extension_pool_test.go

```diff
@@ -76,6 +76,10 @@ func TestAccResourceExtensionPoolBasic(t *testing.T) {
 }
 
 func deleteExtensionPoolWithNumber(startNumber string) error {
+	sdkConfig, err := gcloud.AuthorizeSdk()
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
I would return this error and log it where the function is called from, because the rest of this func won't work if the sdk wasn't authorized correctly